android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -195,6 +195,41 @@ class RNPlayerView(context: ReactApplicationContext) : LinearLayout(context) {
         emitEvent("sourceWarning", it)
     }
 
+    /**
+     * `onCueEnter` event callback.
+     */
+    private val onCueEnter: (PlayerEvent.CueEnter) -> Unit = {
+        emitEvent("cueEnter", it)
+    }
```

Comment:
Ok, I agree about extracting those event handlers to their own components. That would make Android internals a lot easier to maintain/evolve in the future.

But, regarding the established pattern, you're talking about the one inside the Android SDK right? Because this function declaration followed precisely the same pattern as the others.

Or do you want me to change them all in this PR too?

Reply:
Sorry for the confusion.

I meant that what you added in this PR is already established in the code base here. So this is fine to continue.
Changing from `val` to `fun` declarations would have been valid for all occurrences in the file **if** we would like to change it.

Reply:
Oh alright, I see what you meant now, sorry for the misunderstanding. I can surely fix this in another PR, just need to open an issue for it so we can keep track. Thanks for the heads up.